projects/coordinator/src/main/java/org/batfish/coordinator/resources/ContainerResource.java

```diff
@@ -0,0 +1,72 @@
+package org.batfish.coordinator.resources;
+
+import java.nio.file.Files;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.ForbiddenException;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import org.batfish.common.BatfishLogger;
+import org.batfish.common.Container;
+import org.batfish.coordinator.Main;
+
+/**
+ * The {@link ContainerResource} is a resource for servicing client API calls at container level.
+ *
+ * <p>The ContainerResource provides information about a specified container, and provides the
+ * ability to create/delete a specified container for authenticated clients.
```

Comment:
does not have ability to create a container -- that goes somewhere else. (in the ContainersResource)